packages/@react-aria/select/src/useSelect.ts

```diff
@@ -81,8 +83,12 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
     labelElementType: 'span'
   });
 
+  typeSelectProps.onKeyDown = typeSelectProps.onKeyDownCapture;
+  delete typeSelectProps.onKeyDownCapture;
```

Comment:
This will unfortunately cause the `stopPropagation is now the default behavior for events in React Spectrum. You can use continuePropagation() to revert this behavior` error to appear in the console since `typeSelect` has a `stopPropagation` called within. I guess I could add a prop to `useTypeSelect` to suppress the `stopPropagation` call, open to other ideas

Reply:
🙈

```js
if (!('continuePropagation' in event)) {
  event.stopPropagation();
}
```